x-pack/test/api_integration/apis/ml/jobs/jobs_summary.ts

```diff
@@ -0,0 +1,362 @@
+            datafeedId: '',
+            datafeedIndices: [],
+            datafeedState: '',
+            isSingleMetricViewerJob: true,
+            fullJob: {
+              // Only tests against some of the fields in the fullJob property.
+              job_id: SINGLE_METRIC_JOB_CONFIG.job_id,
+              job_type: 'anomaly_detector',
+              description: SINGLE_METRIC_JOB_CONFIG.description,
+              groups: SINGLE_METRIC_JOB_CONFIG.groups,
+              analysis_config: {
+                bucket_span: '15m',
+                detectors: [
+                  {
+                    detector_description: 'mean(responsetime)',
+                    function: 'mean',
+                    field_name: 'responsetime',
+                    detector_index: 0,
+                  },
+                ],
+                influencers: [],
+              },
+            },
+          },
+          {
+            id: MULTI_METRIC_JOB_CONFIG.job_id,
+            description: MULTI_METRIC_JOB_CONFIG.description,
+            groups: MULTI_METRIC_JOB_CONFIG.groups,
+            processed_record_count: 0,
+            memory_status: 'ok',
+            jobState: 'closed',
+            hasDatafeed: false,
+            datafeedId: '',
+            datafeedIndices: [],
+            datafeedState: '',
+            isSingleMetricViewerJob: true,
+          },
+        ],
+      },
+    },
+  ];
+
+  const testDataListNegative = [
+    {
+      testTitle: 'as ML Unauthorized user',
+      user: USER.ML_UNAUTHORIZED,
+      requestBody: {},
+      // Note that the jobs and datafeeds are loaded async so the actual error message is not deterministic.
+      expected: {
+        responseCode: 403,
+        error: 'Forbidden',
+      },
+    },
+  ];
+
+  function compareById(a: { id: string }, b: { id: string }) {
+    if (a.id < b.id) {
+      return -1;
+    }
+    if (a.id > b.id) {
+      return 1;
+    }
+    return 0;
+  }
+
+  function getGroups(jobs: Array<{ groups: string[] }>) {
+    const groupIds: string[] = [];
+    jobs.forEach(job => {
+      const groups = job.groups;
+      groups.forEach(group => {
+        if (groupIds.indexOf(group) === -1) {
+          groupIds.push(group);
+        }
+      });
+    });
+    return groupIds.sort();
+  }
+
+  describe('jobs_summary', function() {
+    before(async () => {
+      await esArchiver.loadIfNeeded('ml/farequote');
+      await ml.testResources.setKibanaTimeZoneToUTC();
+    });
+
+    after(async () => {
+      await ml.api.cleanMlIndices();
+    });
+
+    it('sets up jobs', async () => {
+      for (const job of testSetupJobConfigs) {
+        await ml.api.createAnomalyDetectionJob(job);
+      }
+    });
+
+    for (const testData of testDataListNoJobId) {
+      describe('gets job summary with no job IDs supplied', function() {
+        it(`${testData.testTitle}`, async () => {
+          const { body } = await supertest
+            .post('/api/ml/jobs/jobs_summary')
```

Comment:
Similar to my comment in the `get_field_stats` test: you could define the request code in a separate function in order to avoid code duplication.

Reply:
Done in https://github.com/elastic/kibana/pull/64165/commits/2b63f4504c03ec5d78ccef7cba518a1511be3b15. Note I haven't used a `testDataList` here as the checks for each test have some differences.